Changelog.md

```diff
@@ -29,6 +29,7 @@ Language Features:
 
 
 Compiler Features:
+ * NatSpec: Add support for NatSpec documentation in ``struct`` definition.
```

Comment:
This is the changelog for the already released 0.8.19 version; you're gonna have to move this up to line 10 (after the enum entry).